mobile/lib/src/infrastructure/eeg_data_transformers/baseOpenBCITransformer.dart

```diff
@@ -0,0 +1,56 @@
+import 'dart:async';
+
+abstract class BaseOpenBCITransformer<S, T> implements StreamTransformer<S, T> {
```

Comment:
Pourquoi on fait pas plutot :

```suggestion
abstract class BaseOpenBCITransformer<S, T> implements StreamTransformer<List<S>, List<T>> {
```

Car dans tous les cas, c'est toujours des listes.

Bien sûr, il faut changer du code ailleurs aussi.

Reply:
Je peux ouvrir un issue pour ça et essayer de voir si on peut le faire plus tard, mais j'ai essayé de faire ce changement, et je me retrouve toujours à avoir une incompatiblité dans eeg_repository quand viens le moment de l'utiliser dans la fonction .transform.

Reply:
np